docs/developers/function-file.md

```diff
@@ -2,19 +2,13 @@
 
 Functions files are used to assist fn to help you when creating functions.
 
-The files can be named as:
-
-- func.yaml
-- func.json
-
```

Comment:
intentional change to remove this?

Reply:
We never talk about the json format, so might as well not promote it.